CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## [3.6.1] - TBD
```

Comment:
Let's go with 3.7.0 as the version number since more has changed under the hood than I thought due to the addition of the new `get_partition_values` method in Hive.